# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xpcom/xpcAccessibleValue.cpp
# Commit: dcc478201cb7
# Full Hash: dcc478201cb75d078d3a5fd33f306c7b9cfcd527
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-08-27 03:42:22
# Regressor Bug: 1727643
# File Overlap Count: 1
# Description:
#   Bug 1727643 - P4: Make use of new common methods in platforms and XPCOM. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123698
# ==============================================================================

diff -r 1a9901750064 -r dcc478201cb7 accessible/xpcom/xpcAccessibleValue.cpp
--- a/accessible/xpcom/xpcAccessibleValue.cpp	Thu Aug 26 19:40:05 2021 +0000
+++ b/accessible/xpcom/xpcAccessibleValue.cpp	Thu Aug 26 19:40:06 2021 +0000
@@ -22,12 +22,7 @@
     return NS_ERROR_FAILURE;
   }
 
-  double value;
-  if (Intl()->IsLocal()) {
-    value = Intl()->AsLocal()->MaxValue();
-  } else {
-    value = Intl()->AsRemote()->MaxValue();
-  }
+  double value = Intl()->MaxValue();
 
   if (!IsNaN(value)) *aValue = value;
 
@@ -45,12 +40,7 @@
     return NS_ERROR_FAILURE;
   }
 
-  double value;
-  if (Intl()->IsLocal()) {
-    value = Intl()->AsLocal()->MinValue();
-  } else {
-    value = Intl()->AsRemote()->MinValue();
-  }
+  double value = Intl()->MinValue();
 
   if (!IsNaN(value)) *aValue = value;
 
@@ -68,12 +58,7 @@
     return NS_ERROR_FAILURE;
   }
 
-  double value;
-  if (Intl()->IsLocal()) {
-    value = Intl()->AsLocal()->CurValue();
-  } else {
-    value = Intl()->AsRemote()->CurValue();
-  }
+  double value = Intl()->CurValue();
 
   if (!IsNaN(value)) *aValue = value;
 
@@ -108,12 +93,7 @@
     return NS_ERROR_FAILURE;
   }
 
-  double value;
-  if (Intl()->IsLocal()) {
-    value = Intl()->AsLocal()->Step();
-  } else {
-    value = Intl()->AsRemote()->Step();
-  }
+  double value = Intl()->Step();
 
   if (!IsNaN(value)) *aValue = value;
 
